src/app/app.component.html

```diff
@@ -4,5 +4,28 @@
   License, v. 2.0. If a copy of the MPL was not distributed with this
   file, You can obtain one at https://mozilla.org/MPL/2.0/.
 -->
-
-<p>Triple Banana Browser Share Link</p>
+<div class="bannerContainer" (click)="onBannerClick()">
+  <div class="bannerContent">
+    <div class="bannerIconShadow">
+      <svg width="0" height="0">
+        <defs>
+          <clipPath id="shape120">
+            <path id="shape" class="cls-1"
+              d="M6821,495.533v-4.281c0-1.2-.04-2.4-0.04-3.642a57.7,57.7,0,0,0-.68-7.882,26.144,26.144,0,0,0-2.48-7.483,25.115,25.115,0,0,0-11.04-11.044,26.118,26.118,0,0,0-7.49-2.481,47.28,47.28,0,0,0-7.88-.68c-1.2-.04-2.4-0.04-3.64-0.04h-53.5c-1.2,0-2.4.04-3.64,0.04a57.813,57.813,0,0,0-7.88.68,26.323,26.323,0,0,0-7.49,2.481,25.115,25.115,0,0,0-11.04,11.044,26.144,26.144,0,0,0-2.48,7.483,47.313,47.313,0,0,0-.68,7.882c-0.04,1.2-.04,2.4-0.04,3.642v53.5c0,1.2.04,2.4,0.04,3.641a57.7,57.7,0,0,0,.68,7.883,26.137,26.137,0,0,0,2.48,7.482,25.115,25.115,0,0,0,11.04,11.044,26.261,26.261,0,0,0,7.49,2.481,47.28,47.28,0,0,0,7.88.68c1.2,0.04,2.4.04,3.64,0.04h53.5c1.2,0,2.4-.04,3.64-0.04a57.654,57.654,0,0,0,7.88-.68,26.057,26.057,0,0,0,7.49-2.481,25.115,25.115,0,0,0,11.04-11.044,26.137,26.137,0,0,0,2.48-7.482,47.316,47.316,0,0,0,.68-7.883c0.04-1.2.04-2.4,0.04-3.641V495.533h0Z"
+              transform="translate(-6701 -458)" filter="url(#f1)">
+            </path>
+          </clipPath>
+        </defs>
+      </svg>
+      <img class="bannerIconLarge" src="assets/app_icon.svg">
+    </div>
+    <div class="bannerNameContainer">
+      <h1 class="bannerName bannerLabel">
+        Banana Browser
+      </h1>
+      <h2 class="bannerDescription bannerLabel">
+        {{bannerDescription}}
```

Comment:
Why do we need this dynamic variable? (multilingual support?)

Reply:
If chrome browser I will change the description that "Click here to move link" due to chromium based browser does not allow silent(?) link

Reply:
In my humble opinion, we can make the description show always.
If someone is hurry like me, they might be able to click on it. WDYT?

Reply:
I do want to show Banana browser's advantage(?) even it is short time. If we show the "Click here..." always,
user not using chromium browser, see "click here to move link" but it move to link automatically then he might think "why they show click here?". I think showing advertising description would be better :)

Reply:
I thought as follows:
```
<Icon>   <Subject: Safer, faster, and more stable>
         <Description: Click here to move link> 
```

But if you think dynamic description is better, I trust your choice because your are an expert in this part :)

Thanks!